onnx/helper.py

```diff
@@ -332,6 +333,20 @@ def float32_to_bfloat16(fval: float, truncate: bool = False) -> int:
     return (ival + round) >> 16
 
 
+def bfloat16_to_float32(ival: int) -> Any:
+    if ival == 0x7FC0:
+        return np.float32(np.nan)
+
+    expo = ival >> 7
+    prec = ival - (expo << 7)
+    sign = expo & 256
+    powe = expo & 255
+    fval = float(prec * 2.0 ** (-7) + 1) * 2.0 ** (powe - 127)
+    if sign:
+        fval = -fval
+    return np.float32(fval)
```

Comment:
https://github.com/onnx/onnx/blob/8669fad0247799f4d8683550eec749974b4f5338/onnx/numpy_helper.py#L16
Are they different?

Reply:
I revmoved the one I added and modified the existing one by adding a default value to dims.